tests/conftest.py

```diff
@@ -56,6 +56,7 @@ def start_mock_request(request_mock):
     request_mock.stop()
 
 
+# Used to prevent warning to be displayed during the run of pytest tests
 @pytest.fixture(autouse=True)
```

Comment:
There is only a specific test that should raise these warnings right ? So not sure we want to enable this for every test

Reply:
Well normally it should be only the sdk import tests, however even when they are desactivated, pytest still raise a warning. Not sure where it comes from but it's like pytest run the sdk imports during the tests despite having 0 "from sdk.xx" imports.